### PR TITLE
Module 1 homework

### PR DESCRIPTION
**Pull request by thewildjacko**
Short answers & coding challenge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+a) What does the command ‘git status’ output?</br>
+**It tells you what branch you're in and whether the repo is clean or whether there are uncommitted changes.**
+
+b) In SwiftUI, anything that gets drawn on the screen is a **View**.
+
+c) print(“Hello world”) is an example of a **function** call.
+viewModel.getData() is an example of a **method** call.
+
+d) Name some Views you have seen so far in SwiftUI.
+  - **body**
```
Comment (byaruhaf) [inline README.md]: i would remove 

- **body**
- **ContentView**

File: README.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+f) How do you preview your app in multiple orientations?</br>
+**In Xcode, make sure the canvas is showing (`command-option-return`), and that the preview is running (`command-option-p`). In the bottom left corner of the canvas, click the variants button and select orientation variants from the dropdown menu. Then just click on the orientation you want. While previewing a given orientation, you can mouse over the left and right sides of the canvas and click the arrows that appear to switch between orientations.**
+
+g) An app is made up of **files** of **classes and structs** that contain **properties and methods**.
+
+h) Name two components of a SwiftUI Button.</br>
+**label and action**
+
+i) In git, what is the difference between a local repository and a remote repository?</br>
+**A local repository is stored on your local computer/hard drive, and a remote repository is stored on a server that you can access from anywhere using GitHub**
```
Comment (byaruhaf) [inline README.md:30]: A remote repository is located in the cloud or remote server. it may not be github

**Review by byaruhaf — APPROVED** [review]
Grade:  Above and Beyond
Fantastic work, Jake! The application looks impressive and functions seamlessly, just as anticipated. Kudos!Every answer provided is correct.I've made one minor comments in the pull request (PR).
Your dedication to this assignment is truly commendable.
 I have approved the PR, so please merge it back into the main branch.
I wish you continued success in the upcoming weeks!

File: README.md (hunk 2 above)
Comment (thewildjacko) [reply README.md:30]: Ah good to know, thanks.

**Pull request merged**